canonical-web-and-design/snapcraft.io
# activeDevices module refactor, updated dashboard overview graph

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/1868

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/snaps
- If you have no snaps or all your snaps have no users, the graph should show
- If you have snaps, the graph should show
- If you have a lot of snaps you should get a little loader and the snaps should be added 1 by 1
- Resizing should work
- http://0.0.0.0:8004/<snap_name>/metrics should work as before
